inventory/items/views.py

```diff
@@ -214,11 +252,9 @@ def import_items_csv(request):
                         new_expiry = {
                             "item": current_item.first().id,
                             "expiry_date": row[5],
-                            "quantity": row[3],
+                            "quantity": int(row[3]),
                         }
-                        expiry_serializer = AddItemExpirySerializer(data=new_expiry)
-                        if expiry_serializer.is_valid(raise_exception=True):
```

Comment:
Anyway, is there a need for a timeout functionality because the endpoint seem to accept csv of any file size, then loads indefinitely. Just a point for the future.

Reply:
added a file size limit of 5mb